content/en/docs/concepts/security/service-accounts.md

```diff
@@ -196,6 +196,31 @@ or using a custom mechanism such as an [authentication webhook](/docs/reference/
 You can also use TokenRequest to obtain short-lived tokens for your external application.
 {{< /note >}}
 
+### Restricting access to Secrets {#enforce-mountable-secrets}
+
+Kubernetes provides an annotation called `kubernetes.io/enforce-mountable-secrets`
+that you can add to your ServiceAccounts. When this annotation is applied,
+the ServiceAccount's secrets can only be mounted on specified types of resources,
+enhancing the security posture of your cluster.
+
+You can add the annotation to a ServiceAccount using a manifest:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    kubernetes.io/enforce-mountable-secrets: "true"
+  name: my-serviceaccount
+  namespace: my-namespace
+```
+
+When this annotation is set to "true", the Kubernetes control plane ensures that
+the Secrets from this ServiceAccount are only mountable on the resource kinds
+specified in the annotation value.
```

Comment:
This is technically inaccurate. If the annotation value is `true` it can't also be a list of resource kinds.